src/cc/bpf_module.cc

```diff
@@ -95,6 +98,8 @@ class MyMemoryManager : public SectionMemoryManager {
 
 BPFModule::BPFModule(unsigned flags)
     : flags_(flags), ctx_(new LLVMContext) {
+  InitializeNativeTarget();
```

Comment:
The use case is not in python at all. The use case is for fuse, where the daemon process has no facility to understand the types inside of the maps. Suppose I write to a file, in order to create an entry (key=1, value=2, types are ints):
echo "1 2" > new_entry
My problem statement is how can I have a string-based, simple input/ouput format that works with file-system syntax. The example above is easy enough to implement by hand, but breaks down with structs/unions/nested structs etc. I would like to support those. The implementation seen in this patch is not the best or most portable one, but it lets us prototype such an API on top. Whichever implementation can achieve that I would be happy with. JIT is a fancy trick but just a means to an end. Complex formatted string input is the goal.

Maybe it would help to see a more complete use case?